web/.gitignore

```diff
@@ -3,6 +3,8 @@ output/
 build/
 embedded/
```

Comment:
Is this legacy build folder still needed? Wonder if cleaning this up means we don't need l. 6.